problem1.py

```diff
@@ -0,0 +1,22 @@
+"""
+Question
+Provide an array of n-1 integers in the range of 1 to n with no duplicates 
+in list, One of the integers is missing in the list. 
+Write a code to find the missing integer.
+"""
+"""
+Name: Srinidhi Bhat
+Space Complexity - O(1) - Only variables
+Time Complexity - O(n) - Traversing through the array for getting max, and doing sum
+
```

Comment:
Improve the time complexity. Try binary search

Reply:
Sure Lohith, I will try that and get back to you.